services/web-app/components/markdown/image.module.scss

```diff
@@ -21,3 +25,8 @@
     object-fit: contain;
   }
 }
+
+// targets images on the page not nested inside other components
+:global([class*='layout_column-content']) > .image-container {
```

Comment:
is this what it's supp to look like at small breakpoint? margin left and then take entire width after? feels a bit weird 🤔 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/167061796-cac176e0-f0ad-4cb9-b13c-368331ac7c47.png">

Reply:
Nope, that's weird. I'll fix

Reply:
Updated!